back/app/controllers/web_api/v1/static_pages_controller.rb

```diff
@@ -50,11 +50,13 @@ def update
   end
 
   def destroy
-    page = @page.destroy
+    destroyed = @page.destroy
 
-    if page.destroyed?
-      SideFxStaticPageService.new.after_destroy page, current_user
+    if destroyed
+      SideFxStaticPageService.new.after_destroy destroyed, current_user
       head :ok
+    elsif @page.errors
+      render json: { errors: @page.errors.details }, status: :unprocessable_entity
     else
       head :internal_server_error
```

Comment:
This looks like a very defensive programming habit too me. I cannot think of a way how we would end up in this clause, do you?

If `page.destroy` does not suceed, then we will have `errors` on the record. Let's remove the last `else` clause.

Reply:
We usually do it (search for `head :internal_server_error`), so I would be consistent (whether to refactor everything or nothing). 
Also, if somebody adds `throw :abort` to the model without adding errors, the client will receive 200.

Reply:
@nTraum We prevent page.destroy in the model with an `abort` if it does not have `code: 'custom'`. This pattern is used in a similar way in [custom_field_options_controller.rb](https://github.com/CitizenLabDotCo/citizenlab/blob/c294c7f0fccff3b018e5a5ca96db6cedcd94ff46/back/app/controllers/web_api/v1/custom_field_options_controller.rb#L67)

Reply:
@nTraum I see your point. I'm not sure I follow all the arguments about when/if it might be reached, however (I guess it might also be reached if something goes wrong with a permitted destroy action?).

Since we currently use the pattern:
```
else
   head :internal_server_error
end
```
In more than 30 other places, how about we bring this to next CDM, and discuss the pattern in general (maybe we should remove/change it in other controllers too?), but leave this as it is here for now?

Reply:
Sounds good :+1: